.gitignore

```diff
@@ -5,4 +5,4 @@ dist
 bundle-stats.html
 .vscode
 __diff_output__
-test-results.json
```

Comment:
Looks like this reverts #1361. Is it intended?

Reply:
Oh, wow! This PR was what did it? Great catch! 👍 I was already starting to think we never did anything about this file in the first place

Reply:
Sorry about that :/

Reply:
git blame FTW =)

Reply:
No worries, I’ve actually forgotten to PR it’s removal again